crates/ruff_python_ast/src/typing.rs

```diff
@@ -66,3 +72,48 @@ pub fn is_pep585_builtin(expr: &Expr, context: &Context) -> bool {
         PEP_585_BUILTINS_ELIGIBLE.contains(&call_path.as_slice())
     })
 }
+
+#[derive(is_macro::Is)]
```

Comment:
isn't that a case for `derive(PartialEq, Eq)]`?

Reply:
This macro gives us something slightly different -- we can do `annotation.is_simple()`, similar to what we get with `Result`.